tests/dummy/app/pods/view/renderers/documentation/checkbox-array.md

```diff
@@ -29,3 +29,36 @@ Change what size checkboxes are used. See [ember-frost-core](http://ciena-frost.
   }
 }
 ```
+
+#### renderer.data
+
+Overrides the enum in the model to use specified label and value.
+
+```json
+{
+  "model": "foo",
+  "renderer": {
+    "name": "checkbox-array",
+    "data": [
+      {
+        "label": "The label for the checkbox",
+        "value": "The value for the checkbox"
+      }
+    ]
+  }
+}
+```
+
+#### renderer.selectedValues
+
+Allows checkboxes to be pre-checked. E.g. for retaining checks from query params upon refresh.
+
+```json
+{
+  "model": "foo",
+  "renderer": {
+    "name": "checkbox-array",
+    "selectedValues": ["values", "of", "selected", "checkboxes"]
```

Comment:
I mean the `bunsenValue` you pass into the component, it will have a `foo` property that can have a  value of `['values', 'of', 'selected', 'checkboxes']` just like every other input in a bunsen form. I'm not sure why you want to use some other mechanism for setting the value of something, other than the `value` property.